encoding/encoding.go

```diff
@@ -98,9 +98,6 @@ func (v V2) Encode(ctx context.Context, projectID uint64) string {
 
 func (V2) Decode(accessKey string) (projectID uint64, err error) {
 	parts := strings.Split(accessKey, Separator)
-	if len(parts) < 2 {
-		return 0, ErrInvalidKeyLength
-	}
```

Comment:
I think we still need to check at least against `len < 1` ? Otherwise the below would panic?

Reply:
It's a strings.Split, it should return at least one element (the og string)